SchematicsAPI_Multi_NIC.md

```diff
@@ -0,0 +1,697 @@
+ 2020/10/07 06:39:21 Terraform show |     direction  = "inbound"
+ 2020/10/07 06:39:21 Terraform show |     group      = "r030-d9d66cec-2126-4f0a-bbaa-c0e10cdd35bc"
+ 2020/10/07 06:39:21 Terraform show |     id         = "r030-d9d66cec-2126-4f0a-bbaa-c0e10cdd35bc.r030-589fee55-edef-4b92-925e-6889e6d0e1d1"
+ 2020/10/07 06:39:21 Terraform show |     ip_version = "ipv4"
+ 2020/10/07 06:39:21 Terraform show |     remote     = "0.0.0.0/0"
+ 2020/10/07 06:39:21 Terraform show |     rule_id    = "r030-589fee55-edef-4b92-925e-6889e6d0e1d1"
+ 2020/10/07 06:39:21 Terraform show | 
+ 2020/10/07 06:39:21 Terraform show |     tcp {
+ 2020/10/07 06:39:21 Terraform show |         port_max = 22
+ 2020/10/07 06:39:21 Terraform show |         port_min = 22
+ 2020/10/07 06:39:21 Terraform show |     }
+ 2020/10/07 06:39:21 Terraform show | }
+ 2020/10/07 06:39:21 Terraform show | 
+ 2020/10/07 06:39:21 Terraform show | # ibm_is_security_group_rule.vnf_sg_rule_in_all:
+ 2020/10/07 06:39:21 Terraform show | resource "ibm_is_security_group_rule" "vnf_sg_rule_in_all" {
+ 2020/10/07 06:39:21 Terraform show |     direction  = "inbound"
+ 2020/10/07 06:39:21 Terraform show |     group      = "r030-d9d66cec-2126-4f0a-bbaa-c0e10cdd35bc"
+ 2020/10/07 06:39:21 Terraform show |     id         = "r030-d9d66cec-2126-4f0a-bbaa-c0e10cdd35bc.r030-f43da5dd-0c59-4330-80e1-e19975b3fa41"
+ 2020/10/07 06:39:21 Terraform show |     ip_version = "ipv4"
+ 2020/10/07 06:39:21 Terraform show |     remote     = "0.0.0.0/0"
+ 2020/10/07 06:39:21 Terraform show |     rule_id    = "r030-f43da5dd-0c59-4330-80e1-e19975b3fa41"
+ 2020/10/07 06:39:21 Terraform show | }
+ 2020/10/07 06:39:21 Terraform show | 
+ 2020/10/07 06:39:21 Terraform show | # ibm_is_security_group_rule.vnf_sg_rule_out_all:
+ 2020/10/07 06:39:21 Terraform show | resource "ibm_is_security_group_rule" "vnf_sg_rule_out_all" {
+ 2020/10/07 06:39:21 Terraform show |     direction  = "outbound"
+ 2020/10/07 06:39:21 Terraform show |     group      = "r030-d9d66cec-2126-4f0a-bbaa-c0e10cdd35bc"
+ 2020/10/07 06:39:21 Terraform show |     id         = "r030-d9d66cec-2126-4f0a-bbaa-c0e10cdd35bc.r030-0a2826c7-829d-4d93-82d6-1c05688b6752"
+ 2020/10/07 06:39:21 Terraform show |     ip_version = "ipv4"
+ 2020/10/07 06:39:21 Terraform show |     remote     = "0.0.0.0/0"
+ 2020/10/07 06:39:21 Terraform show |     rule_id    = "r030-0a2826c7-829d-4d93-82d6-1c05688b6752"
+ 2020/10/07 06:39:21 Terraform show | }
+ 2020/10/07 06:39:21 Terraform show | 
+ 2020/10/07 06:39:21 Terraform show | 
+ 2020/10/07 06:39:21 Terraform show | Outputs:
+ 2020/10/07 06:39:21 Terraform show | 
+ 2020/10/07 06:39:21 Terraform show | map_instance = "r030-88b35711-82aa-45ae-b0e6-12785e7c2b55"
+ 2020/10/07 06:39:21 Command finished successfully.
+ 2020/10/07 06:39:23 Done with the workspace action
+
+</code></pre>
+
+### 7. Assign Floating-point IP to the F5 VSI created in Step 7.
```

Comment:
please mention about the name and target id. the name is any string to denote the fip and the target id should be the primary network interface id obtained from the output for previous step(6)